### PR TITLE
Generate_package fixes

### PR DESCRIPTION
**Pull request by Adrianzatreanu**

### CODE REVIEW

File: bears/generate_package.py [hunk 1]
```diff
@@ -51,8 +52,11 @@ def create_file_structure_for_packages(root_folder, file_to_copy, object_name):
     upload_package_folder = os.path.join(root_folder, object_name, object_name)
     os.makedirs(upload_package_folder, exist_ok=True)
     touch(os.path.join(upload_package_folder, '__init__.py'))
-    shutil.copyfile(file_to_copy, os.path.join(upload_package_folder,
-                                               object_name + '.py'))
+    try:
+        shutil.copyfile(file_to_copy, os.path.join(upload_package_folder,
+                                                   object_name + '.py'))
+    except shutil.SameFileError:
+        pass
```
Comment (Adrianzatreanu) [inline bears/generate_package.py]: this is here because since i changed the new versioning system, some bears fail to upload saying that its the same file or whatever, so I need this until i find out why some bears are geting uploaded multiple times.

**Comment by Adrianzatreanu** [comment]
i have absolutely no idea how to test for those branches, so im leaving the PR for review until i miraculously find and ill remove the pragmas then :D

**Comment by fneu** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/591/commits/ad63a7fc420a4584a735a4a78e2fcf796a2b9b37 https://github.com/coala-analyzer/coala-bears/pull/591/commits/aa51f58444502f603e15ccd08818acac41354387

File: bears/generate_package.py [hunk 2]
```diff
@@ -64,9 +65,9 @@ def perform_register(path, file_name):
     """
     subprocess.call(
         [sys.executable, 'setup.py', 'sdist', 'bdist_wheel'], cwd=path)
-    subprocess.call(['twine', 'register', '-r', 'pypitest', os.path.join(
```
Comment (fneu) [inline bears/generate_package.py]: if this is unrelated, maybe put it in an own commit?

File: bears/generate_package.py [hunk 3]
```diff
@@ -101,7 +102,8 @@ def main():
 
     os.makedirs(os.path.join('bears', 'upload'), exist_ok=True)
 
-    bear_file_name_list = glob('bears/**/*Bear.py')
```
Comment (fneu) [inline bears/generate_package.py]: this also really doesn't look like a versioning change ;)

**Comment by fneu** [comment]
looks like there are tests, can't you check with `coverage html` which lines/branches are missing and add them somehow to a test or add another similar test? If you need help, try to find out who worked on the tests before (i.e. `git blame`) and ask them.

**Comment by Adrianzatreanu** [comment]
@fneu you were right, i changed to different commits :)

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/591/commits/767cf143b8f7edd05dffa9c52e9f766c6e41f92a

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/591/commits/cad76bebfc96516d2ae22a3486f7702440938e3b

**Comment by sils1297** [comment]
unack https://github.com/coala-analyzer/coala-bears/pull/591/commits/3ae7ce8a085d72f3f2236e81d49e9b290c877eb1

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/591/commits/2b50597f40a836999d5006b1e021792bb3ea7308

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/591/commits/111a588d5f546fc5d4ceb283e5857b353a695dfc ?

**Comment by Adrianzatreanu** [comment]
reack 1748a39

**Comment by Adrianzatreanu** [comment]
@rultor merge
